RDS Pricing info

svdgraaf opened the pull request:
As also was requested in #105 here is a first implementation of RDS pricings.
Be aware that I've added `requests` to the requirements.

You can see an example here: http://svdgraaf.nl/ec2instances.info/rds.html

You can run the parser all by itself (it will download the latest RDS data from the pricing api), or you can feed it the data file itself (faster debugging):

```
$ python rds.py /tmp/rds-index.json
```

After that, you can just run the regular `fab render_html` command to render the rds pages as well:

```
$ fab render_html
Loading data from www/instances.json...
Rendering to www/index.html...
Loading data from www/rds.json...
Rendering to www/rds.html...

Done.
```

Known issues
* For some reason some pricings are missing from the RDS pricing index (eg: check MariaDB reserved pricings). I've filed an issue with AWS for this.

TODO:
* The template doesn't show all the details from the json yet. 
* Also, I haven't parsed the storage and network data, but that's easily added later on.

svdgraaf commented:
Hmmm, I've found a price calculation error, on it

svdgraaf commented:
Fixed it, I had a minor bug in the RDS reserved pricing calculations, all good now :)

svdgraaf commented:
I added a lot more columns to the output

powdahound commented:
This is great. Would be nice to expand the `fab build` command to do the RDS-specific scraping so that the update process stays the same.

svdgraaf commented:
I extended the fab:build step, it now also fetches the rds data. Be aware that the output of aws is around 26MB at the moment, so it might take some time. The parsing is pretty fast afterwards :)

I also rebased on master, and rebuild the index files.

nicka commented:
:+1:

svdgraaf commented:
I added tabs to the navigation, and moved the twitter / last updated label to the right of the page (to make room). You can see it in action on: http://rdsinstances.info/

tvb commented:
:1 Awesome job @svdgraaf!

powdahound commented:
Great stuff here. Did you look into sharing the common parts of the .mako templates like the header/tabs/social buttons/footer? Seems like that'd be the last nice change to have so the two sites can coexist nicely and not diverge.

nicka commented:
@svdgraaf Fix it :)

svdgraaf commented:
@nicka thanks for the headsup!
@powdahound I added a `base.mako` file, which is re-used by the ec2 and rds pages.

btw: I have no experience with the Mako templating engine, so if you have any suggestions, please let me know :)

powdahound commented:
I've done some rough work to make this work with the latest changes in master and merged it in. See the above commit for more details.

It's not in the cleanest state right now but I fear that this PR will just keep getting more stale over time so wanted to get it merged in while it was reasonable. I'll be opening new issues to track some problems I encountered while doing this work.

As far as the domain goes, I think having rdsinstances.info redirect to ec2instances.info/rds/ is the cleanest solution since 1) we can keep serving everything out of one S3 bucket 2) it keeps the local dev setup clean. Let me know if you'd like to work out a transfer of the domain so it can be managed it in the same location as ec2instances.info

Thanks for your help getting this started!
